### DISCUSSION TITLE
WSCV-DT1 (Telemundo 51) [US]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#355] [Show and tell]
### Channel ID

WSCVDT1.us

### Stream URL

https://video-weaver.mad01.hls.ttvnw.net/v1/playlist/CoYFC0ZSeb7P7jhrf2PgMFsUeURgHDZ8N9yX5bO4YFP13_DClcgmV_LDHVwJcKoqQa5VyEGz38Wz8Re7fy3SzySB1v40jp88ZGf7wQHjQO2yWW89Q7FUzfCo2kHbcgB8yYigC4NVVCNg0-Kf1NjmJHrHmUnxQTukeGB9mLzRD6tjERxDahsXXBLN2lNsWwp2BzVIjfDFXF6VHtGIjW6IqMimKt9UtTzqlxuAY5k4Rf2RJdxUkAB6O87UpiomNUQxxqBeo-vSx4lBxrbb5oEdlXWxOa5ewjEGxY1FyEb2O2Z0yroTq8cR24vLTTtkQHSnW2TE5SmZmnXXOvO8XZo3U2JnAbfgM7iZC2rvflJJe3lOtpKnlGM3gwcCD4EPVUYDkM9QBqesEUUUBs0mGl3vtF83Y58eI3s2eXbMf6HO2QBUYoEF8YynL8gst8PwvA619jegux3xdA5I4kWCwN9K49X8ErKb4VeJhGWRF19Vxn2rqnuj-iDYLMB_BCz8ub4qfZS9Bx34VO78AgcirardeBZJMbk1eLZ8dPogFeNgQfrjjZULi5nNtHuEvsp-7OTGp-VhaKztrQj3SxjUCnF-kXOHTHPI-B_vZVq3AVQxHd5tYyFDOr-5w3skZ2Su2_0QEQhACExP-OBc5Kwf6tFvheS2gVZOtNBJYHbWiYnM_qP0RLqHGmlNeyHm0MOCa-kPs55GRKV38ZEh_nVOsVyNd_TQ0H22hWgxUsn_h65sTG-lkgzCMv3Vw_zKK-oZlFR5FIb2m_zqMIh2YYyqOsmZwWWD5mACYAUVVUJ1_kxMqiJM835zsDoE9as4HD8sZz_FITC3ss2AzNEEaDwrx7P_m6BBhiJBXoS1iBoMuwO-eJCVJZ507lTrIAEqCWV1LXdlc3QtMjDOBA.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Dead stream: please close.

**Comment by freearhey** [comment]
https://video-weaver.mad01.hls.ttvnw.net/v1/playlist/CoYFC0ZSeb7P7jhrf2PgMFsUeURgHDZ8N9yX5bO4YFP13_DClcgmV_LDHVwJcKoqQa5VyEGz38Wz8Re7fy3SzySB1v40jp88ZGf7wQHjQO2yWW89Q7FUzfCo2kHbcgB8yYigC4NVVCNg0-Kf1NjmJHrHmUnxQTukeGB9mLzRD6tjERxDahsXXBLN2lNsWwp2BzVIjfDFXF6VHtGIjW6IqMimKt9UtTzqlxuAY5k4Rf2RJdxUkAB6O87UpiomNUQxxqBeo-vSx4lBxrbb5oEdlXWxOa5ewjEGxY1FyEb2O2Z0yroTq8cR24vLTTtkQHSnW2TE5SmZmnXXOvO8XZo3U2JnAbfgM7iZC2rvflJJe3lOtpKnlGM3gwcCD4EPVUYDkM9QBqesEUUUBs0mGl3vtF83Y58eI3s2eXbMf6HO2QBUYoEF8YynL8gst8PwvA619jegux3xdA5I4kWCwN9K49X8ErKb4VeJhGWRF19Vxn2rqnuj-iDYLMB_BCz8ub4qfZS9Bx34VO78AgcirardeBZJMbk1eLZ8dPogFeNgQfrjjZULi5nNtHuEvsp-7OTGp-VhaKztrQj3SxjUCnF-kXOHTHPI-B_vZVq3AVQxHd5tYyFDOr-5w3skZ2Su2_0QEQhACExP-OBc5Kwf6tFvheS2gVZOtNBJYHbWiYnM_qP0RLqHGmlNeyHm0MOCa-kPs55GRKV38ZEh_nVOsVyNd_TQ0H22hWgxUsn_h65sTG-lkgzCMv3Vw_zKK-oZlFR5FIb2m_zqMIh2YYyqOsmZwWWD5mACYAUVVUJ1_kxMqiJM835zsDoE9as4HD8sZz_FITC3ss2AzNEEaDwrx7P_m6BBhiJBXoS1iBoMuwO-eJCVJZ507lTrIAEqCWV1LXdlc3QtMjDOBA.m3u8 (offline)
<img width="815" alt="image" src="https://user-images.githubusercontent.com/7253922/232234799-4f7ade40-835d-49a3-8159-f14934631629.png">
